packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -498,7 +498,7 @@ const Tooltip = React.forwardRef(function Tooltip(inProps, ref) {
   const handleRef = useForkRef(children.ref, handleFocusRef);
 
   // There is no point in displaying an empty tooltip.
-  if (title === '') {
+  if (!title) {
```

Comment:
```suggestion
  if (!title || title === undefined) {
```

Add undefined

Reply:
I don't think so, `!undefined` returns true so the condition applies. I don't think you need the `title === undefined` part.

Reply:
Okay

Reply:
Unfortunately, this would also catch the `title={0}` case. You'd have to be more specific here.
Also, please add some tests to verify this behavior.

Reply:
I don't know why the previous demo shows `title={0}` in a tooltip.
Please refer to this demo:
https://codesandbox.io/s/gracious-blackburn-p9j2f2?file=/demo.tsx

Reply:
> I don't know why the previous demo shows title={0} in a tooltip.
Please refer to this demo:
https://codesandbox.io/s/gracious-blackburn-p9j2f2?file=/demo.tsx

I am confused. I agree with @michaldudak, you need to exclude the `number` condition:

```js
if (typeof title !== 'number' && !title){
}
```

Reply:
Sorry, I misunderstand this. Thank you @siriwatknp.